.github/workflows/workflow-yaml-check.yml

```diff
@@ -25,6 +25,13 @@ jobs:
       - name: Install yamllint
         run: pip install yamllint
 
+      - name: Checkout code
+        uses: actions/checkout@v4
```

Comment:
checking out the entire directory seems excessive when we can just point to the .yamllint.yml file?

Reply:
conversations should be resolved by the reviewer.

please refer to: 

https://github.com/orgs/community/discussions/63863

Yes, there is special handling for how myworkflow.yaml should reference script.js. When you call a reusable workflow, GitHub Actions will copy the contents of the workflow file to the current working directory. This means that the script.js file will not be available in the current working directory unless you explicitly reference it.

To reference the script.js file, you can use the uses keyword. The uses keyword allows you to import a workflow file from another repository. In your case, you would use the following syntax:

```
uses: calleerepo/.github/workflows/script.js@main
```

Reply:
I will wait until we meet and you explain your comprehension of the reference you've given. I don`t think we are trying to achieve the same thing.

Reply:
Yes, after reviewing this I see that we are not able to refer to the file from the called repository github-workflows directory - or at least I couldn't find a way.